client/app/views/partials/messageconsole.html

```diff
@@ -25,12 +32,18 @@
       </div>
 
       <div data-ng-switch-when="13">
-        <span data-translate>Resource can only be accessed via the Tor network</span>
+        <span data-translate
+          >Resource can only be accessed via the Tor network</span
+        >
       </div>
 
       <div data-ng-switch-when="14">
         <span data-translate>The upload request exceeds the size limit</span>
       </div>
+
+      <div data-ng-switch-when="17">
+        <span data-translate>A user with this username already exists</span>
+      </div>
```

Comment:
Please revise the changes to this file including only the addition of error file 17 and exclude every refactor when proposing a specific patch.